internal/config/config_util.go

```diff
@@ -112,3 +113,7 @@ func ListCacheTtlSecsToDuration(secs int64) time.Duration {
 
 	return time.Duration(secs * int64(time.Second))
 }
+
+func DefaultMaxDownloadParallelism() int {
+	return max(16, 2*runtime.NumCPU())
```

Comment:
Shouldnt the default come after testing. Do we want this to be default now?

Reply:
As discussed, I want to introduce the dependency on CPU cores, we can change to a better value based on our testing.

Reply:
is this function a well known function to depend on it. is it in a standard package which will work in all VM's?

Reply:
Resolving for now based on chat comments that its a standard package in go